### PR TITLE
feat: Add reports about live agent load

### PR DESCRIPTION
**Pull request by aswindevps**
## Description

This change adds a new report page for live agent load which shows the below metrics.

1. Open conversations - Total, Unattended, and Unassigned conversations count.
2. Agent status - Online, Busy and Offline agents count.
3. Conversations by agents - Table of agents with Total and Unattended open conversations count.

Demo : https://loom.com/share/6feb00bd27a241938e85253ac9a57547

Fixes #[chatwoot/product/issues/364](https://github.com/chatwoot/product/issues/364)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/reports/components/overview/MetricCard.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    margin: var(--space-smaller);
+    background: rgba(37, 211, 102, 0.1);
+    color: var(--g-400);
+    font-size: var(--font-size-mini);
+
+    .ellipse {
+      background-color: var(--g-400);
+      height: var(--space-smaller);
+      width: var(--space-smaller);
+      border-radius: 50%;
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/reports/components/overview/MetricCard.vue:71]:
```suggestion
      border-radius: var(--border-radius-rounded);
```

File: app/javascript/dashboard/helper/actionCable.js [hunk 2]
```diff
@@ -145,6 +147,11 @@ class ActionCableConnector extends BaseActionCableConnector {
   onNotificationCreated = data => {
     this.app.$store.dispatch('notifications/addNotification', data);
   };
+
+  // eslint-disable-next-line no-unused-vars
+  onFirstReplyCreated = data => {
+    bus.$emit('fetch_overview_reports');
+  };
 }
```
Comment (muhsin-k) [inline app/javascript/dashboard/helper/actionCable.js:155]:
```suggestion
  onFirstReplyCreated = () => {
    bus.$emit('fetch_overview_reports');
  };
}
```

File: app/javascript/dashboard/i18n/locale/en/report.json [hunk 3]
```diff
@@ -381,5 +381,33 @@
         "TOOLTIP": "Total number of responses / Total number of CSAT survey messages sent * 100"
       }
     }
+  },
+  "OVERVIEW_REPORTS": {
+    "HEADER": "Overview",
+    "LIVE": "live",
```
Comment (fayazara) [inline app/javascript/dashboard/i18n/locale/en/report.json:387]:
```suggestion
    "LIVE": "Live",
```

**Review by fayazara — COMMENTED** [review]
Tested locally, looks good. Have left a small comment

**Comment by iamsivin** [comment]
@aswindevps Please fix broken builds.

File: app/javascript/dashboard/helper/actionCable.js (hunk 2 above)
Comment (aswindevps) [reply app/javascript/dashboard/helper/actionCable.js]: Done

File: app/javascript/dashboard/routes/dashboard/settings/reports/components/overview/MetricCard.vue (hunk 1 above)
Comment (aswindevps) [reply app/javascript/dashboard/routes/dashboard/settings/reports/components/overview/MetricCard.vue]: Done

**Review by sojan-official — APPROVED** [review]
Ruby code LGTM

**Review by muhsin-k — APPROVED** [review]

**Review by iamsivin — APPROVED** [review]

**Pull request merged**